src/main/webapp/course.html

```diff
@@ -222,13 +228,13 @@ <h1 class="p-4 text-center">Add Your Review</h1>
             <div class="col">
               <p class="lead">Want to Translate to English?</p>
               <select class="custom-select mb-5" id="translate">
-                <option value="false" disabled selected>Wants to translate?</option>
+                <option value="false" disabled selected>
+                  Wants to translate?
```

Comment:
Here it is "want" to translate, it is now fixed in master but just wanted to let u know